gateway/pgrest/connections/connections.go

```diff
@@ -91,18 +95,31 @@ func (c *connections) Upsert(ctx pgrest.OrgContext, conn pgrest.Connection) erro
 	if conn.Status == "" {
 		conn.Status = pgrest.ConnectionStatusOffline
 	}
+
+	if conn.AccessSchema == "" {
+		if conn.Type == "database" {
+			conn.AccessSchema = "enabled"
+		} else {
+			conn.AccessSchema = "disabled"
+		}
```

Comment:
There's not need for the else statement here, this is more cleaner and easier to read:

```go
conn.AccessSchema = "disabled"
if conn.Type == "database" {
  conn.AccessSchema = "enabled"
}
```